.github/workflows/ci.yml

```diff
@@ -563,4 +563,37 @@ jobs:
       - name: Run integration tests for cockroach
         working-directory: internal/integration
         run: go test -race -count=2 -v -run="Cockroach" -version="cockroach" ./...
-  
\ No newline at end of file
+  
+  integration-spanner:
+    runs-on: ubuntu-latest
+    services:
+      spanner:
+        image: gcr.io/cloud-spanner-emulator/emulator:1.4.6
+        env:
+          SPANNER_EMULATOR_HOST: localhost:9010
+        ports:
+          - 9010:9010
+          - 9020:9020
```

Comment:
Why the second port is necessary?

Reply:
It's not, this just mirrors what `gcloud emulators spanner start` sets up. We can pull the extra port.